### ISSUE TITLE
Should linked logotypes in headers fulfill 2.5.3 Label in Name?

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#2011]
> Success Criterion 2.5.3 Label in Name (Level A): For user interface components with labels that include text or images of text, the name contains the text that is presented visually.

Let's say my logo has a stylized version of ACME on it. It is clearly an [image of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text) as defined in the definition. However, in the actual SC [1.4.5 Images of Text](https://www.w3.org/TR/WCAG21/#images-of-text) (Not confusing _at all_!), logotypes are exempt.

If I have the following HTML, do I need to include ACME into the alternative text because it is also a visible text? Currently, I lean towards “yes”, but the difference in handling logotypes across several SCs (usually they are exempt) wants me to double check.

```html
<a href="/">
   <img alt="home" src="logotype.svg">
</a>
```

**Comment by yatil** [comment]
> This has been discussed previously: [#722 (comment)](https://github.com/w3c/wcag/issues/722#issuecomment-490876875)

Thanks, @alastc! I tried to search for it, but I did not find anything.

**Comment by patrickhlauke** [comment]
although this issue is already closed, i'd still add that 

> in the actual SC 1.4.5 Images of Text (Not confusing at all!), logotypes are exempt.

yes, it's exempt from the requirement there to use actual styled HTML text (if possible) rather than an image. so 1.4.5's ask doesn't apply here, nothing more, nothing less.

the fact that 2.5.3 mentions images of text is to make it independent of 1.4.5, i'd say. like yes, if you're using a (non-exempt) image of text, you fail 1.4.5, but regardless your accessible name for it also needs to include the visible text (otherwise you ALSO fail 2.5.3)

**Comment by detlevhfischer** [comment]
I think this has to be considered in view of user expectations. Often, graphic logos have an additional small strapline with some generic marketing lingo like "paving the way", "imagine the future", "the people company" etc. which may be part if the same graphic. I guess not many speech users would think of using such suplementary text to activate the element - but it may be the only visible text in the logo. That is one reason why I lean towards generally exemptig text in logos from 2.5.3.

**Comment by jake-abma** [comment]
My recollection is that the standpoint of the WCAG WG is that all info in a logo is always exempted (period).

Has same kind of questions this year in multiple issues and this was the outcome.

**Comment by yatil** [comment]
> My recollection is that the standpoint of the WCAG WG is that all info in a logo is always exempted (period).

@alastc’s link [above](https://github.com/w3c/wcag/issues/2011#issuecomment-904623910) to issue https://github.com/w3c/wcag/issues/722#issuecomment-490876875 actually shows that this is not the consensus. (And I agree and did expect that. Sometimes you just need a validity check.)

(And thanks @patrickhlauke to clarify my snarky comment. This is also how I understand it. :-) )

**Comment by patrickhlauke** [comment]
Not sure it's as cut and dry as that. Certainly there are specific aspects where logos are exempt (e.g. text and non-text contrast), but not a blanket exemption.

**Comment by patrickhlauke** [comment]
I would still expect "paving the way" to be part of the text alternative/1.1.1. so it would require extra effort NOT to have that as part of the accessible name. I don't see a particular reason to exempt this sort of visible text as part of a logo.

**Comment by detlevhfischer** [comment]
Arguably "ACME  home page" would be a better alt for 1.1.1 than ACME paving the way home page"? I would not fail 1.1.1 for dropping marketing fluff from the alt text of a logo. See all that lovely subjectivity again.
